addons/l10n_ar/models/account_move.py

```diff
@@ -357,11 +362,36 @@ def _l10n_ar_get_invoice_totals_for_report(self):
             tax_lines=tax_line_vals_list,
         )
 
+        temp = self.tax_totals
         if include_vat:
-            temp = self.tax_totals
             tax_totals['amount_total'] = temp['amount_total']
             tax_totals['formatted_amount_total'] = temp['formatted_amount_total']
 
+        # RG 5614/2024: Show ARCA VAT and Other National Internal Taxes
+        if self.l10n_latam_document_type_id.code in ['6', '7', '8']:
+
+            # Prepare the subtotals to show in the report
+            detail_info = {
+                'vat_taxes': {"name": _("VAT Content %s", self.currency_id.symbol), "tax_amount": 0.0},
+                'other_taxes': {"name": _("Other National Ind. Taxes %s", self.currency_id.symbol), "tax_amount": 0.0},
+            }
+            for item in temp['groups_by_subtotal'][_('Untaxed Amount')]:
+                if item.get('tax_group_id') in nat_internal_other_taxes:
+                    detail_info['other_taxes']["tax_amount"] += item['tax_group_amount']
+                elif item.get('tax_group_id') in vat_taxes:
+                    detail_info['vat_taxes']["tax_amount"] += item['tax_group_amount']
+
+            # Clean up 0.0 values, we dont want it to print them in the report
+            for item in list(detail_info.keys()):
+                if detail_info[item]["tax_amount"] <= 0.0:
+                    detail_info.pop(item)
+
+            # Format the amounts to show in the report
+            for _item, values in detail_info.items():
+                values.update({"formatted_amount": formatLang(self.env, values["tax_amount"])})
```

Comment:
```suggestion
                values.update({"formatted_amount_tax": formatLang(self.env, values["tax_amount"])})
```
Maybe let's keep the same naming convention.